projects/canopy/src/lib/details/details-panel-heading/details-panel-heading.component.ts

```diff
@@ -20,13 +22,17 @@ let nextUniqueId = 0;
 export class LgDetailsPanelHeadingComponent {
   @Input() headingLevel;
   @Input() isActive = false;
+  @Input() showIcon = true;
```

Comment:
I think true is ok 🤔  because all old details components should have the default 'generic' variant meaning they won't have an icon. I guess this property is more of an escape hatch for if people are doing something a bit different

Reply:
Yeah, fair enough